src/GitVersion.Core.Tests/VersionCalculation/BaseVersionCalculatorTests.cs

```diff
@@ -18,15 +18,12 @@ public class BaseVersionCalculatorTests : TestBase
         public void ChoosesHighestVersionReturnedFromStrategies()
         {
             var dateTimeOffset = DateTimeOffset.Now;
-            var versionCalculator = GetBaseVersionCalculator(contextBuilder =>
-            {
-                contextBuilder.OverrideServices(services =>
-                {
-                    services.RemoveAll<IVersionStrategy>();
-                    services.AddSingleton<IVersionStrategy>(new V1Strategy(DateTimeOffset.Now));
-                    services.AddSingleton<IVersionStrategy>(new V2Strategy(dateTimeOffset));
-                });
-            });
+            var versionCalculator = GetBaseVersionCalculator(contextBuilder => contextBuilder.OverrideServices(services =>
```

Comment:
I'd prefer to keep the curly braces here as it's more readable and aligned

Reply:
Sorry I missed this formatting issue. Just to be sure what's your preference between

1.
```csharp
var versionCalculator = GetBaseVersionCalculator(contextBuilder =>
    contextBuilder.OverrideServices(services =>
    {
        services.RemoveAll<IVersionStrategy>();
        services.AddSingleton<IVersionStrategy>(new V1Strategy(DateTimeOffset.Now));
        services.AddSingleton<IVersionStrategy>(new V2Strategy(dateTimeOffset));
    }));
```

2.
```csharp
var versionCalculator = GetBaseVersionCalculator(contextBuilder =>
{
    contextBuilder.OverrideServices(services =>
    {
        services.RemoveAll<IVersionStrategy>();
        services.AddSingleton<IVersionStrategy>(new V1Strategy(DateTimeOffset.Now));
        services.AddSingleton<IVersionStrategy>(new V2Strategy(dateTimeOffset));
    }
}));
```

In case you go with choice 2, we will probably have to either reduce the severity, prevent preference of expression body for lambdas or use pragam disable for such places.

Reply:
option 1 seems fine if the second expression is on new line and aligned as in the sample